reporters/allure-testinfo-adapter.js

```
@@ -0,0 +1,27 @@
+// reporters/allure-testinfo-adapter.js
+const { allure } = require('allure-playwright');
+
+/** @type {import('@playwright/test/reporter').Reporter} */
+class AllureTestInfoAdapter {
+  onTestBegin(test, testInfo) {
```

The reporter should verify the presence of `testInfo.allure` and fail loudly when it's missing. The corrections state: **"HIGH: Do not silently no-op when testInfo.allure is missing. Fail fast or emit a loud error/warning to surface misconfiguration; ensure tests use testInfo.allure.* so labels are correctly attached to the current test."**

Suggested approach: in `onTestBegin`, if `!testInfo.allure` then throw an informative Error explaining that the Allure Playwright reporter is not configured (so CI/locally the failure is obvious). Do not provide a silent fallback or no-op proxy.